src/Parser/Druid/Feral/Modules/FeralCore/Snapshot.js

```diff
@@ -42,30 +40,31 @@ class Snapshot extends Analyzer {
   };
 
   // extending class should fill these in:
-  spellCastId;
-  debuffId;
-  isProwlAffected;
-  isTigersFuryAffected;
-  isBloodtalonsAffected;
-  durationOfFresh;
+  spellCastId = null;
+  debuffId = null;
+  isProwlAffected = false;
+  isTigersFuryAffected = false;
+  isBloodtalonsAffected = false;
+  durationOfFresh = false;
```

Comment:
Since it never changes, how about making it static instead?

Reply:
Please avoid changing types of vars as much as possible. A better default value for this would be `null`.